### PR TITLE
[#975562] localStorage is not cross domain, so ensure we cannot check ju...

### PR DESCRIPTION
**Pull request by ScottDowne**
...st it inside verify

### CODE REVIEW

File: webmaker-auth-client.js [hunk 1]
```diff
@@ -254,10 +254,6 @@
 
       self.verify = function () {
 
-        if (self.storage.get()) {
```
Comment (cadecairos) [inline webmaker-auth-client.js:257]: if we're removing this feature, perhaps we should remove all localStorage related code? There's no use for it anymore if we don't get the instant log-in from localStorage.

File: webmaker-auth-client.js [hunk 2]
```diff
@@ -307,7 +276,7 @@
 
         };
 
-        http.send(body);
+        http.send(null);
```
Comment (cadecairos) [inline webmaker-auth-client.js]: does `XMLHttpRequest` need null passed in?

Comment (ScottDowne) [inline webmaker-auth-client.js]: Think so, I'll double check though.

Comment (ScottDowne) [inline webmaker-auth-client.js]: Does not need. Removing and landing.

**Pull request merged**